seatunnel-connectors-v2/connector-elasticsearch/src/main/java/org/apache/seatunnel/connectors/seatunnel/elasticsearch/serialize/ElasticsearchRowSerializer.java

```diff
@@ -82,65 +83,82 @@ public String serializeRow(SeaTunnelRow row) {
     private String serializeUpsert(SeaTunnelRow row) {
```

Comment:
Could you add a test case for this method? Thanks.

Reply:
ACK